thirdeye/thirdeye-frontend/app/pods/components/detection-health/template.hbs

```diff
@@ -1,84 +1,103 @@
-<ul class="te-horizontal-cards__single-card">
-  <li class="te-horizontal-cards__card-title">
-    <label for="select-dimension" class="control-label te-label--no-bottom">
-      30-day status
+<div class="te-horizontal-cards te-content-block__detection-health">
```

Comment:
It's better to use semantic tag here, such as `section`

Reply:
Sure.